osbs/conf.py

```diff
@@ -426,3 +426,18 @@ def get_oauth2_token(self):
     def get_reactor_config_secret(self):
         return self._get_value("reactor_config_secret", self.conf_section,
                                "reactor_config_secret")
+
+    def get_client_config_secret(self):
+        return self._get_value("client_config_secret", self.conf_section,
+                               "client_config_secret")
+
+    def get_token_secrets(self):
+        value = self._get_value("token_secrets", self.conf_section,
+                                "token_secrets")
+        token_dict = {}
+        if value:
+            for pairs in value.split():
+                pair = pairs.split(':',1)
+                token_dict[pair[0]] = pair[1] if len(pair)>1 else ''
```

Comment:
should there be allowed more than 1 ':' ?

yeah I can raise exception when it is obviously wrong